routes/api/index.js

```diff
@@ -0,0 +1,7 @@
+const router = require("express").Router();
+const medRoutes = require("./med");
+
+// Med routes
+router.use("/med", medRoutes);

```

Comment:
Use router.use("/api", medRoutes); if you want an /api/something structure. You currently have /med/something as the api route